mypy.ini

```diff
@@ -6,3 +6,42 @@ strict_optional = True
 warn_unused_ignores = True
 warn_redundant_casts = True
 warn_unused_configs = True
+
```

Comment:
I'd like to see a comment block around here explaining the incremental whitelist strategy for posterity/for new contributors who go looking in this config file.

Reply:
done!